src/main/java/cz/fi/muni/pa165/esports/entity/MatchRecord.java

```diff
@@ -19,6 +20,16 @@
     private int score;
     private long match_number;
 
+    /*
+    @OneToMany(mappedBy = "match")
+    private Set<Team> teams = new HashSet<>();
+
+    @OneToMany(mappedBy = "match")
+    private Set<Player> players = new HashSet<>();
+
+    @ManyToOne
+    private Competition competition;
+    */
     @ManyToOne
     private Team team;
 
```

Comment:
Please add a `MatchRecord` constructor that takes only `id` (so hibernate works well).
Location in code is near other constructors, github won't allow me to comment where I want to...

Technically, we don't need the constructor that takes name, etc, but you can keep it :)

Reply:
Also, `MatchRecord` doesn't have a `name` attribute as per class diagram, so we should remove it and methods that reference it as well.

Reply:
In class diagram, the `match_number` is now called `matchNumber` -- we changed that yesterday during the call, please also update in code (just use `Shift+F6` in Idea to rename all occurences).